microsoft/testsuites/core/hv_module.py

```diff
@@ -234,24 +244,29 @@ def reload_hyperv_modules(self, case_name: str, log: Logger, node: Node) -> None
                 "kernel and therefore can not be reloaded"
             )
 
-        result = node.execute(
+        node.execute(
             ("for i in $(seq 1 %i); do " % loop_count)
             + f"modprobe -r -v {module}; modprobe -v {module}; "
             "done; sleep 1; "
             "ip link set eth0 down; ip link set eth0 up; dhclient eth0",
             sudo=True,
             shell=True,
+            nohup=True,
         )
+        pgrep = node.tools[Pgrep]
+        while pgrep.get_processes("nohup"):
```

Comment:
Is that possible to have multiple nohup processes? How about return the nohup process, if nohup=True?

Reply:
nohup won't be called by the OS as far as I know, it's for remote users running commands. It's possible there could be a lingering nohup process from another test if it hasn't been torn down correctly though. That's a sticky problem 🤔
Thank you for the note, I've marked this as a draft and I'm going to run some tests to verify this works as expected.

Reply:
during run reload module and down and up eth0 in the background, the pgrep may fail to execute.